tests/testthat/test-operators.R

```diff
@@ -91,3 +91,17 @@ test_that("t() is aliased to edge reversal for graphs", {
   expect_that(vcount(t(g)), equals(vcount(g)))
   expect_that(as_edgelist(t(g)), equals(as_edgelist(g)[, c(2, 1)]))
 })
+
+test_that("vertices() works", {
+  g <- make_empty_graph(1)
+
+  g_all_unnamed <-  g + vertices("a", "b")
+  expect_s3_class(V(g_all_unnamed), "igraph.vs")
+  expect_identical(V(g_all_unnamed)$name, c( NA, "a", "b"))
+
+  g_mix_named_unnamed <- g + vertices("a", "b", foo = 5)
```

Comment:
What happens with `foo = 6:7` and `foo = 8:10` ?

Reply:
I added a test. The error also happens with v1.6.0 and seems logical.